src/templates/meetup.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import { graphql } from 'gatsby'
+import rehypeReact from 'rehype-react'
+
+import Layout from 'src/components/layout'
+import { TopSection } from 'src/components/layout/Section'
+import Grid, { Col, Row } from 'src/components/layout/Grid'
+import { H2, H3, H5, P } from 'src/components/text'
+import { Card } from 'src/components/elements'
+import Header from 'src/components/layout/Header'
+import {
+  UpcomingTrainingSection,
+  selectTrainingById,
+  selectUpcomingTrainings,
+} from 'src/components/training'
+import { PaymentSection } from 'src/components/payment'
+import { Link, Breadcrumb } from 'src/components/navigation'
+import { MEETUP } from 'src/config/data'
+
+const renderAst = new rehypeReact({
+  createElement: React.createElement,
+  components: {
+    p: P,
+    h2: H2,
+    h3: H3,
+    h5: H5,
+  },
+}).Compiler
+
+const MeetUpPage = ({ data }) => {
+  const { city, instanceID } = data.markdownRemark.frontmatter
```

Comment:
@ric9176 can we get the id from the file name instead since it's there? We can add to the query:
```
query {
  markdownRemark {
      frontmatter {
        instanceID
        city
        paragraphs
      }
      htmlAst
      fileAbsolutePath
    }
}
```
then ` const { fileAbsolutePath } = data.markdownRemark`
then we can use the `path` from Nodejs (https://nodejs.org/api/path.html#path_path_basename_path_ext) and do  `path.basename(fileAbsolutePath, path.extname(fileAbsolutePath))`